lab-dan/lib/fileRead.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {};
+
+const fs = require('fs');
+
+module.exports.splice = function (pathArray, printcb, next) {
+
+  if (!Array.isArray(pathArray)) console.error('function was not passed an array of paths');
```

Comment:
console.error is a solid way to dev, but as soon as you're done with development you can migrate those to actually `throw` an error.